src/dotnet/UI.Blazor.App/Services/AppServiceStarter.cs

```diff
@@ -151,6 +152,7 @@ public async Task AfterFirstRender(CancellationToken cancellationToken)
             Services.GetRequiredService<AppPresenceReporter>().Start();
             Services.GetRequiredService<AppIconBadgeUpdater>().Start();
             Services.GetService<RpcPeerStateMonitor>()?.Start(); // Available only on the client
+            Services.GetRequiredService<ContactSync>().Start();
```

Comment:
Does it make sense to start it somewhere else than Maui?

Reply:
yes, since there is also greeting logic.